OPERATING.md

```diff
@@ -0,0 +1,161 @@
+# Operating: data transformation and ingest for VEDA
+
+## Data
+
+### `collections/`
+
+Holds collection jsons
+
+Should follow the following format:
+
+```json
+{
+    "id": "<collection-id>",
+    "type": "Collection",
+    "links":[
+    ],
+    "title":"<collection-title>",
+    "extent":{
+        "spatial":{
+            "bbox":[
+                [
+                    "<min-longitude>",
+                    "<min-latitude>",
+                    "<max-longitude>",
+                    "<max-latitude>",
+                ]
+            ]
+        },
+        "temporal":{
+            "interval":[
+                [
+                    "<start-date>",
+                    "<end-date>",
+                ]
+            ]
+        }
+    },
+    "license":"MIT",
+    "description": "<collection-description>",
+    "stac_version": "1.0.0",
+    "dashboard:is_periodic": "<true/false>",
+    "dashboard:time_density": "<month/>day/year>",
+    "item_assets": {
+        "cog_default": {
+            "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+            "roles": [
+                "data",
+                "layer"
+            ],
+            "title": "Default COG Layer",
+            "description": "Cloud optimized default layer to display on map"
+        }
+    }
+}
+
+```
+
+### `events/`
+
+Holds jsons of events passed to the ingestion pipeline.
+Can either be a single event or a list of events.
+
+Should follow the following format:
+
+```json
+{
+    "collection": "<collection-id>",
+    "discovery": "<s3/cmr>",
+
+    ## for s3 discovery
+    "prefix": "<s3-key-prefix>",
+    "bucket": "<s3-bucket>",
+    "filename_regex": "<filename-regex>",
+    "datetime_range": "<month/day/year>",
+    
+    ## for cmr discovery
+    "version": "<collection-version>",
+    "temporal": ["<start-date>", "<end-date>"],
+    "bounding_box": ["<bounding-box-as-comma-separated-LBRT>"],
+    "include": "<filename-pattern>",
```

Comment:
Maybe a nested json like: 
```
{
  "cmr": {
    "version": "<collection-version>",
    ...
  },
  "s3": {
    "prefix": "<s3-key-prefix>",
    ...
  },
  "collection": "<collection-id>"
}
```
would be better for this?

Reply:
+1